Userland/Services/LookupServer/LookupServer.cpp

```diff
@@ -77,24 +79,36 @@ LookupServer::LookupServer()
 
 void LookupServer::load_etc_hosts()
 {
-    m_etc_hosts.clear();
-    auto add_answer = [this](Name const& name, RecordType record_type, DeprecatedString data) {
-        m_etc_hosts.ensure(name).empend(name, record_type, RecordClass::IN, s_static_ttl, move(data), false);
+    auto new_hosts_or_error = this->try_load_etc_hosts();
+    if (new_hosts_or_error.is_error())
+        dbgln("Ignoring '/etc/hosts', keeping old values");
+    else {
```

Comment:
Nit: braces on all branches if they're on one.

Reply:
Background: I was experimenting with other things and inserted a few `dbgln` temporarily in the else branch, and apparently forgot to remove the braces. Whoops!